Minebot/src/main/java/net/famzangl/minecraft/minebot/ai/AIHelper.java

```diff
@@ -88,11 +87,7 @@
 	private HashMap<KeyType, KeyboardInputController> keys = new HashMap<KeyType, KeyboardInputController>();
 	
 	protected boolean doUngrab;
-
-	protected MapReader activeMapReader;
```

Comment:
I intented to reactivate the map reader and stats some day (=> I use it for debugging to see what the bot is doing a lot)

Reply:
Ah, I didn't see it used / have never used it in my time with Minebot, and so I was just removing the "bulk" code, same with the stats. It's easily restored I think.